### ISSUE TITLE
Abnormal serum Insulin-like growth factor binding protein 3 level

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11371]
**Preferred term label:**

**Synonyms**
Abnormal serum IGFBP-3 level
Abnormal IGF binding protein-3 level

**Definition (free text, please give PubMed ID)**
PMID: 37008935 The GH binds to a GHR, which activates JAK2 and promotes the phosphorylation of different members of the signal transducer. The final result is the IGF-1 production. IGF-1 binds to **IGFBP-3** or IGFBP-5 and the acid-labile subunit (ALS) and constitutes the ternary complex.
(...) Thereafter, IGF-1 is primarily secreted into the blood and then links to one of six high-affinity IGF-1 binding proteins (IGFBPs). For 75–90%, IGF-1 is linked to **IGFBP-3**, which is the most abundant among the IGFBPs family, and for 1%, circulates unbound. The linkage between IGF-1 and IGFBP-3 constitutes a complex called the “binary complex”. Soon after, this binary complex is stabilized by the linking to an acid-labile subunit (ALS), codified by the gene IGFALS into the liver. The ternary complex reduces IGF-1 and IGF-2 circulating levels and increases their half-life.

**Parent term (use hpo.jax.org/app)**
IGFBP-3 is part of the ternary complex, along with IGF-1 and the acid-labile subunit (ALS). Therefore, it is part of the growth hormone axis, but it is not a hormone in itself.
Perhaps it would be more accurate to include it within the term "Abnormal growth hormone axis," or within "Abnormal circulating protein concentration" (HP:0010876).
If necessary, I can create "Abnormal growth hormone axis" or suggest modifying the term "Abnormal growth hormone level" (HP:0032367) to "abnormal growth hormone axis" to include abnormal levels of IGFBP-3 and ALS along with GH.

- Abnormal circulating protein concentration HP:0010876
- Abnormal growth hormone level HP:0032367

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:633 Laron syndrome
ORPHA:220465 Laron syndrome with immunodeficiency 

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Let's put this for now simply under hormone. The HPO is not trying to model pathophysiology, but rather observations.

Adding new term: 
- ID: HP:6001444
- label: Abnormal circulating insulin-like growth factor 3 concentration
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: PMID:37008935
- def: The concentration of IGFBP-3 (Insulin-like Growth Factor-Binding Protein 3) in the blood circulation is outside the limits of normal.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11371
- synonyms: 


I also added a "decreased" term for Laron syndrome:

Adding new term: 
- ID: HP:6001445
- label: Decreased circulating insulin-like growth factor 3 concentration
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: PMID:1373347
- def: The concentration of IGFBP-3 (Insulin-like Growth Factor-Binding Protein 3) in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11371
- synonyms:
